src/features/users/usersSlice.ts

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-param-reassign */
```

Comment:
Avoid disabling ESLint rules unless it is absolutely necessary. In this case, no-param-reassign can be avoided by returning a new state instead of directly mutating the existing one.